docs/ldml/tr35-numbers.html

```diff
@@ -2074,6 +2074,7 @@ <h2><a name="Currencies" href="#Currencies" id="Currencies">4
     present. See the main document <i>Section 4.1 <a href=
     "tr35.html#Multiple_Inheritance">Multiple
     Inheritance</a></i>.</p>
+    <p><span class="changed"><strong>Note: </strong>If a currency decimal pattern does not contain a currency symbol placeholder, then the currency symbol should be placed where the decimal symbol would otherwise be placed.</span></p>
```

Comment:
This seems like it should be stronger / more prominent than a "Note".  This will become a requirement for a conforming implementation.

Reply:
It's a "should" note, so that doesn't put a requirement on conforming implementations. But I'm happy to change it if you have suggestions. 
(For one thing, we should clarify whether it is a should for the cldr production data vs for implementations of formatters (vs others?))

Reply:
I think it's pretty clear that this is a "must" requirement.  If you encounter a currency pattern without \u00A4, where are you supposed to put the symbol?  It is wrong to just leave it out.  I don't think a conforming CLDR implementation should be allowed to ignore this rule.